File: app/src/main/java/com/itachi1706/cheesecakeutilities/Objects/DualLineString.java
package com.itachi1706.cheesecakeutilities.Objects;

/**
 * Created by Kenneth on 29/4/2017.
 * for com.itachi1706.cheesecakeutilities.Objects in CheesecakeUtilities
 */

public class DualLineString {

    private String main, sub;

    public DualLineString(String main, String sub) {
        this.main = main;
        this.sub = sub;
    }

    public String getMain() {
        return main;
    }

    public void setMain(String main) {
        this.main = main;
    }

    public String getSub() {
        return sub;
    }

    public void setSub(String sub) {
        this.sub = sub;
    }
}

File: app/src/main/java/com/itachi1706/cheesecakeutilities/objects/DualLineString.kt
package com.itachi1706.cheesecakeutilities.objects

/**
 * Created by Kenneth on 29/4/2017.
 * Modified to Kotlin on 24/11/2017.
 * for com.itachi1706.cheesecakeutilities.Objects in CheesecakeUtilities
 */

data class DualLineString(var main: String?, var sub: String?, var extra: Any?) {
    constructor(main: String?, sub: String?): this(main, sub, null)
}
